libcxx/include/__algorithm/ranges_starts_with.h

```diff
@@ -50,14 +54,36 @@ struct __fn {
       _Pred __pred   = {},
       _Proj1 __proj1 = {},
       _Proj2 __proj2 = {}) {
-    return __mismatch::__fn::__go(
+    if constexpr (sized_sentinel_for<_Sent1, _Iter1> && sized_sentinel_for<_Sent2, _Iter2>) {
```

Comment:
Can you confirm that we already have test coverage for these new branches being introduced? Do we check with the appropriate types of sentinels and iterators already in our tests?

Reply:
I manually inserted `return false` in each branch and verified that all branches are covered. Do we have a code coverage generator for `libc++` test cases?

Reply:
Great, thanks! We don't have a code coverage generator right now. I think it's kinda tricky to do right with template instantiations but it could be worthwhile to have something that works well, if that's possible.